src/StateService/Verification/VerificationContext.cs

```diff
@@ -0,0 +1,139 @@
+using Akka.Actor;
+using Neo.Cryptography;
+using Neo.Cryptography.ECC;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Network.P2P;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.Plugins.StateService.Storage;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using Neo.Wallets;
+using System;
+using System.Collections.Generic;
+
+namespace Neo.Plugins.StateService.Verification
+{
+    public class VerificationContext
+    {
+        private const uint MaxValidUntilBlockIncrement = 100;
+        private StateRoot root;
+        private ExtensiblePayload payload;
+        private readonly Wallet wallet;
+        private readonly KeyPair keyPair;
+        private readonly int myIndex;
+        private readonly uint rootIndex;
+        private readonly ECPoint[] verifiers;
+        private int M => verifiers.Length - (verifiers.Length - 1) / 3;
+        private readonly Dictionary<int, byte[]> signatures = new Dictionary<int, byte[]>();
```

Comment:
```suggestion
using Akka.Actor;
using Neo.Cryptography;
using Neo.Cryptography.ECC;
using Neo.IO;
using Neo.Ledger;
using Neo.Network.P2P;
using Neo.Network.P2P.Payloads;
using Neo.Persistence;
using Neo.Plugins.StateService.Storage;
using Neo.SmartContract;
using Neo.SmartContract.Native;
using Neo.Wallets;
using System;
using System.Collections.Concurrent;

namespace Neo.Plugins.StateService.Verification
{
    public class VerificationContext
    {
        private const uint MaxValidUntilBlockIncrement = 100;
        private StateRoot root;
        private ExtensiblePayload payload;
        private readonly Wallet wallet;
        private readonly KeyPair keyPair;
        private readonly int myIndex;
        private readonly uint rootIndex;
        private readonly ECPoint[] verifiers;
        private int M => verifiers.Length - (verifiers.Length - 1) / 3;
        private readonly ConcurrentDictionary<int, byte[]> signatures = new ConcurrentDictionary<int, byte[]>();
```

Use concurrent?

Reply:
Why use concurrent, it is only in one actor.

Reply:
`VerificationService` use Concurrent for his dictionary and it use this class

Reply:
I don't think we need concurrent in `VerificationService`, do we?

Reply:
Done.